src/main/java/com/cloudogu/smeagol/repository/domain/Name.java

```diff
@@ -0,0 +1,58 @@
+package com.cloudogu.smeagol.repository.domain;
+
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+/**
+ * Name is simple string value object, which can never be empty.
+ */
+public final class Name {
+
+    private final String value;
+
+    private Name(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the name.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a Name from a string.
+     *
+     * @param value string a name
+     *
+     * @return name
+     */
+    public static Name valueOf(String value) {
+        checkArgument(!Strings.isNullOrEmpty(value), "name can not be null or empty");
+        return new Name(value);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)